### PR TITLE
Force browse version control directories

### PR DESCRIPTION
**Pull request by AndiDog**
It's a security hazard to configure web servers to serve `.git` and the like, so find them with the "Forced Browse" scan.

P.S. Why does this file exist twice?

### CODE REVIEW

**Comment by thc202** [comment]
The duplicated files are now deleted, thanks for pointing that out.
You have to update the PR, sorry ;)

I guess the changes could be applied to the other "directorylist" files as well?
I wonder if we should add to some place that the files were changed?

**Comment by thc202** [comment]
Do you plan to change lower case version of "directorylist"?
Could you change the commit message of the second commit from "Fix..." to "Normalise..."?
The `changes` in `ZapAddOn.xml` do not mention Bazaar.

**Comment by AndiDog** [comment]
Also added the strings to lowercase lists ("CVS" wasn't necessary there, already included) and renamed the commit message.

File: src/org/zaproxy/zap/extension/directorylistv2_3/files/fuzzers/dirbuster/directory-list-2.3-big.txt [hunk 1]
```diff
@@ -74249,6 +74253,7 @@ hclogo
 buton
 test3
 CVSROOT
+CVS
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/directorylistv2_3/files/fuzzers/dirbuster/directory-list-2.3-big.txt]: (adding comment again to show properly in PR :/ )
It seems that this is already included (in line 6356).

**Comment by AndiDog** [comment]
Now should hopefully be unique (I grepped "cvs" and "CVS" for duplicates).

File: src/org/zaproxy/zap/extension/directorylistv2_3/ZapAddOn.xml [hunk 2]
```diff
@@ -7,6 +7,7 @@
 	<changes>
 	<![CDATA[
 	Removed repeated files.<br>
+	Added strings for version control directories of Git, Mercurial, SVN, CVS, Bazaar.<br>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/directorylistv2_3/ZapAddOn.xml]: The CVS mention should be dropped.

**Comment by thc202** [comment]
Thanks!
How would you like to be credited?
https://github.com/zaproxy/zap-core-help/wiki/HelpCredits

**Comment by AndiDog** [comment]
You can add me as "Andreas Sommer" wherever it makes sense. Thanks! :)

**Comment by thc202** [comment]
Done in zaproxy/zap-core-help@87a3a69
Thank you!

**Pull request merged**